README.md

```diff
@@ -60,12 +60,14 @@ metadata:
 spec:
   llm:
     providers:
+    - type: openai
     - credentialsSecretRef:
         name: openai-api-keys
       models:
       - name: gpt-3.5-turbo
       name: openai
       url: https://api.openai.com/v1
+    - type: bam
```

Comment:
@Nehanth bam is not a supported type through operator. supported ones are azure_openai;bam;openai;watsonx

Reply:
changed with watsonx

Reply:
@xrajesh in that case should we remove it from the [enum](https://github.com/openshift/lightspeed-operator/blob/main/api/v1alpha1/olsconfig_types.go#L226) as well?

Reply:
@vbelouso - good point - BAM is a IBM internal service , our users will not have access to that - But we will continue to use it for our testing , but we dont want to show that as a supported one , as users will not be able to get access.We can keep it until we use it for tests (ols service) use it.